Lesson25/Anil/filter.js

```diff
@@ -0,0 +1,428 @@
+		price: 10.99,
+	},
+	{
+		title: "Harry Potter and the Sorcerer's Stone",
+		author: 'J.K. Rowling',
+		genre: 'Fantasy',
+		price: 9.99,
+	},
+];
+
+const cheapBooks = (array,price) => [...array].filter((book)=> book.price < 10 );
+console.log("cheapbooks:", cheapBooks(booksExercise9,10) ) ;
+
+// Exercise 4:
+// Given an array of user objects, filter out the users who are below 25 years old.
+const usersExercise4 = [
+	{
+		id: 1,
+		name: 'John',
+		age: 25,
+		email: 'john@example.com',
+	},
+	{
+		id: 2,
+		name: 'Alice',
+		age: 32,
+		email: 'alice@example.com',
+	},
+	{
+		id: 3,
+		name: 'Bob',
+		age: 28,
+		email: 'bob@example.com',
+	},
+	{
+		id: 4,
+		name: 'Emily',
+		age: 21,
+		email: 'emily@example.com',
+	},
+	{
+		id: 5,
+		name: 'Michael',
+		age: 35,
+		email: 'michael@example.com',
+	},
+	{
+		id: 6,
+		name: 'Sara',
+		age: 29,
+		email: 'sara@example.com',
+	},
+	{
+		id: 7,
+		name: 'David',
+		age: 42,
+		email: 'david@example.com',
+	},
+	{
+		id: 8,
+		name: 'Olivia',
+		age: 31,
+		email: 'olivia@example.com',
+	},
+	{
+		id: 9,
+		name: 'Daniel',
+		age: 27,
+		email: 'daniel@example.com',
+	},
+	{
+		id: 10,
+		name: 'Sophia',
+		age: 23,
+		email: 'sophia@example.com',
+	},
+];
+
+const under25 = (array,age) => [...array].filter((person)=> person.age < 25 );
+console.log("under25:", under25(usersExercise4,25) ) ;
+
+
+// Exercise 5:
+// Given an array of sentences, filter out the sentences that have more than 10 words.
+const sentencesExercise5 = [
+	'The quick brown fox jumps over the lazy dog.',
+	'I love to code and solve problems.',
+	'The sun sets in the west.',
+	'Life is like a box of chocolates.',
+	'The cat meowed loudly.',
+	'She walked along the beach, enjoying the salty breeze.',
+	'The rain poured down, soaking everything in sight.',
+	'He smiled and waved goodbye.',
+	'The mountain peaks were covered in snow.',
+	'The room was filled with laughter and joy.',
+	'I have been dreaming of traveling the world and exploring new cultures and cuisines.',
+	'The city lights shimmered in the night sky, creating a mesmerizing spectacle.',
+	'She diligently practiced the piano for hours, striving for perfection in every note.',
+	'The ancient ruins stood as a testament to a long-lost civilization, captivating the imaginations of visitors.',
+	'As the waves crashed against the shore, a sense of tranquility washed over me.',
+	'In the depths of the forest, hidden treasures await those who dare to seek them.',
+	'The starry night sky stretched endlessly, a canvas of beauty and mystery.',
+	'With each step, the hiker felt closer to the summit, fueled by determination and a sense of adventure.',
+	'The aroma of freshly brewed coffee permeated the air, enticing passersby to indulge in a cup of warmth.',
+	'The butterfly gracefully fluttered its wings, showcasing vibrant colors in a delicate dance.',
+];
+
+const under10Words = (array,length) => [...array].filter((sentence)=> sentence.split(" ").length < 10 );
```

Comment:
Since you pass a parameter to the function, use it 🙂
```suggestion
const under10Words = (array, length) => [...array].filter((sentence)=> sentence.split(" ").length < length );
```